pkg/logger/logger.go

```diff
@@ -20,6 +20,15 @@ const (
 	LogLevelWarning LogLevel = "Warning"
 )
 
+// logLevelOrder defines the order of log levels from most verbose to least verbose
+var logLevelOrder = map[LogLevel]int{
+	LogLevelTrace:   0,
+	LogLevelDebug:   1,
+	LogLevelInfo:    2,
+	LogLevelWarning: 3,
+	LogLevelOff:     4,
```

Comment:
This is interesting. Do we expose this option as well? Let's expose it and update docs.

Reply:
I confirmed it's documented: https://atmos.tools/cli/configuration/#logs